contrib/ovirt/lib/ovirtlago/testlib.py

```diff
@@ -132,13 +132,13 @@ def options(self, parser, env=None):
     def configure(self, options, conf):
         super(LogCollectorPlugin, self).configure(options, conf)
 
-    def addError(self, test, err):
-        self._addFault(test, err)
+    def addError(self, test):
```

Comment:
This is part of the nose plugin interface, it might be called by nose with that extra err parameter :/

    http://nose.readthedocs.org/en/latest/plugins/interface.html#nose.plugins.base.IPluginInterface.addError

You might remove it from _addFault though, there's no need for it there